src/test/resources/mappings/getBooking_A9999AC.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
This prisoner data was missing, resulting in an index of only 24 - should be 25.